service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -236,7 +268,9 @@ private void createNewMethod(
       PostMethodRequest postMethodRequest,
       WorkflowDescription workflowDescription,
       List<MethodInputMapping> methodInputMappings,
-      List<MethodOutputMapping> methodOutputMappings)
+      List<MethodOutputMapping> methodOutputMappings,
+      GithubMethodDetails githubMethodSourceDetails,
```

Comment:
small nit: can this variable also be renamed to `githubMethodDetails` to keep it consistent?